### DISCUSSION TITLE
How to integrate KeePassXC to AdsPower?

### DISCUSSION THREAD

**Discussion by ibordun** [keepassxreboot/keepassxc#10662] [Q&A] [answered]
Hello!
I'm trying to integrate KeePassXC to AdsPower via extension, but something went wrong...
AdsPower is antidetect browser. https://www.adspower.com

KeePassXC - unknown
KeePassXC-Browser - 1.9.0.3
Operating system: Win64
Browser: Chrome/Chromium 10

<< snip >> No PDF!
7.0.0.0

**Comment by varjolintu** [comment, marked as answer]
Use the Custom Browser feature in KeePassXC's Browser Integration settings. You'll have to provide the correct Native Messaging Folder manually to the option. The location depends on the browser.

**Reply by ibordun** [reply]
I've tried to provide the correct Native Messaging Folder manually, but I didn't find the correct one. How can I find it?

**Reply by varjolintu** [reply]
https://github.com/keepassxreboot/keepassxc-browser/wiki/Troubleshooting-guide#find-the-native-messaging-configuration-path-with-strace
